pkg/model/resource/resource.go

```diff
@@ -54,13 +55,14 @@ type Resource struct {
 
 	// CRDVersion holds the CustomResourceDefinition API version used for the Resource.
 	CRDVersion string `json:"crdVersion,omitempty"`
-	// WebhookVersion holds the {Validating,Mutating}WebhookConfiguration API version used for the Resource.
+
+	// WebhookVersion holds the {Validating,Mutating}WebhookConfiguration API version used for the Data.
```

Comment:
```suggestion
	// WebhookVersion holds the {Validating,Mutating}WebhookConfiguration API version used for the Resource.
```
The automatic replacement of `Resource` to `Data` also changed this but in this case it is refering to the `Resource` type.